### PR TITLE
#2022 - Form.io definitions cache update for releases

### PR DESCRIPTION
**Pull request by ann-aot**
- version added to form name
_screenshots_
image

### CODE REVIEW

File: sources/packages/backend/libs/utilities/src/config/config.service.ts [hunk 1]
```diff
@@ -245,4 +245,12 @@ export class ConfigService {
       process.env.KEYCLOAK_AUTH_URL,
     ).href;
   }
+
+  /**
+   * Gets the current API version.
+   * @returns current API version.
+   */
+  get getAPIVersion(): string {
```
Comment (dheepak-aot) [inline sources/packages/backend/libs/utilities/src/config/config.service.ts:253]: It is already a getter. So the name could be just apiVersion. `get apiVersion()`

**Review by dheepak-aot — APPROVED** [review]
Nice work @ann-aot . One minor comment.

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by sh16011993 — APPROVED** [review]
LGTM @ann-aot  👍 . Nice work. Thank You for the doubt clarification via code walkthrough.

**Pull request merged**